src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String [] args) {
+        Machine [] machin = {
```

Comment:
name it machines or machineArr

Reply:
machineArray,I wasn't thinking straight.